anime-list-master.xml

```diff
@@ -28700,6 +28700,12 @@
   <anime anidbid="10430" tvdbid="259675" defaulttvdbseason="0" episodeoffset="1" tmdbid="" imdbid="tt3222236">
     <name>Uchuu Senkan Yamato 2199: Hoshi-Meguru Hakobune</name>
   </anime>
+  <anime anidbid="10607" tvdbid="259675" defaulttvdbseason="0" episodeoffset="" tmdbid="" imdbid="tt3689894">
+    <name>Uchuu Senkan Yamato 2199: Tsuioku no Koukai</name>
+    <mapping-list>
+      <mapping anidbseason="1" tvdbseason="0">;1-1;</mapping>
+    </mapping-list>
```

Comment:
No need for this mapping-list, `defaulttvdbseason=0` already maps AniDB S1 to TVDB S0